### ISSUE TITLE
Dumping schema: `Cannot create schema: no namespace for types found`

### ISSUE THREAD

**Issue by smilesrg** [thecodingmachine/graphqlite-bundle#158]
It seems there's a problem with the `"symfony/dependency-injection": "^5.4"` and the bundle while dumping schma using ` bin/console graphqlite:dump-schema`:

```
In SchemaFactory.php line 395:
                                                                               
  Cannot create schema: no namespace for types found (You must call the SchemaFactory::addTypeNamespace() at least once). 
```

ref: https://github.com/thecodingmachine/graphqlite/issues/541

**Comment by smilesrg** [comment]
It seems that compiler pass is not proceed for some reason

**Comment by smilesrg** [comment]
Even extension `GraphQLiteExtension` seems not being loaded

**Comment by JacynoAdam** [comment]
This seems to be the change causing it:
https://github.com/symfony/dependency-injection/commit/58f2988128d2d278280781db037677a32cf720db#diff-9d584e939647584064ecaad604456078cfda5f765f85f7928a3bd0a131f6b169L448

Bringing back the `continue` on `$edge->isLazy()` (which as far as I can tell just reverses all the changes) seems to fix it, though I'm not familiar with Symfony internals so I don't really know what's the exact cause here.

**Comment by smilesrg** [comment]
@JacynoAdam do you think the `graphqlite:dump-schema` command can be fixed somehow?

**Comment by JacynoAdam** [comment]
@smilesrg For now I just version locked `symfony/dependency-injection` to version 5.4.16, might add a patch that just reverts that one commit later on, to keep updates but also have the command working. 

I'm just hoping someone will fix it in this Bundle or on Symfony's side in the near future.

**Comment by andrew-demb** [comment]
Did you check if it works with 5.4.18?

This issue seems to be related to the problem https://github.com/symfony/symfony/issues/48895

**Comment by smilesrg** [comment]
Thanks for your help! After downgrading to `symfony/dependency-injection:5.4.16` the error has been fixed. @andrew-demb there is no `5.4.18` version released yet, however, upgrading to `5.4.x-dev` also fixed the error
